### ISSUE TITLE
[Screen Reader – Make Code - Create a project] Visual label and programmatic label are not same for 'Give your project a name' edit field under 'Create a project' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3561]
**User Experience**:
If visual label of some controls is different from the aria label defined for them, the screen reader dependent user will get confused regarding the name of the controls. Thus, the user may get disoriented.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing under "My Project" section in Home page and select 'New Project' Button.
4. Navigate all controls under 'Create a Project' pop-up.
5. Open developer tool using F12 key and navigate to ‘Give your project a name' edit field.
6. Verify the issue.

**Actual Result:**
Visual label and programmatic label are not same for 'Give your project a name' edit field under 'Create a project' pop-up.

**Expected Result:**
Visual label and programmatic label should be same for 'Give your project a name' edit field under 'Create a project' pop-up, so that screen reader could read it correctly.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 2.5.3 - Label in Name
](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, NVDA, JAWS

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.5.3, #Win10-Edge(Chromium)

![WCAG2 5 3_Create a project_Visual label and programmatic label are not same for 'Give your project a name' edit field](https://user-images.githubusercontent.com/72871089/97958658-aa748500-1dd3-11eb-950a-5d8bc8bf129e.jpg)
